### DISCUSSION TITLE
Windows URL Path

### DISCUSSION THREAD

**Discussion by Safman999** [dadoonet/fscrawler#1646] [Q&A]
Hi,
    I index Windows file servers which works great, rather than drive letters we use the format of "\\servername\share"  A double back slash to begin with (for some reason it is showing as a single on this page)

Everything works well including OCR however we really need to capture the Windows path to the file e.g. "\\servername\test\fox.txt"
in that format (so double \ to begin then servername and \ share )
Is there an easy way to do this? current path as seen in Elastic of "file:////server//ocr/bill.png"  means we have limitations when we need to show a Windows path to end users (they have access to the shared folder and prefer to copy and paste a Windows URL in explorer to open the file)

Hope this makes sense

Regards
Saf

**Comment by Safman999** [comment]
Fixed

**Reply by dadoonet** [reply]
Great! How did you fix it?
